src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -267,15 +277,14 @@ typedef struct {
  */
 
 typedef struct {
-    DOMString url;                        //!< For local candidates this is the URL of the ICE server from which the candidate was obtained
-    DOMString transportId;                //!< Not used currently. ID of object that was inspected for RTCTransportStats
-    CHAR address[IP_ADDR_STR_LENGTH + 1]; //!< IPv4 or IPv6 address of the candidate
-    DOMString protocol;                   //!< Valid values: UDP, TCP
-    DOMString relayProtocol;              //!< Protocol used by endpoint to communicate with TURN server. (Only for local candidate)
-                                          //!< Valid values: UDP, TCP, TLS
-    INT32 priority;                       //!< Computed using the formula in https://tools.ietf.org/html/rfc5245#section-15.1
-    INT32 port;                           //!< Port number of the candidate
-    DOMString candidateType;              //!< Type of local/remote ICE candidate
+    DOMString url;                               //!< For local candidates this is the URL of the ICE server from which the candidate was obtained
```

Comment:
Is it necessary to use DOMString here, or would it be possible to use a character array like CHAR address[IP_ADDR_STR_LENGTH + 1] instead?

Reply:
I believe we want to follow the spec linked in the comment above this struct definition:
https://www.w3.org/TR/webrtc-stats/#icecandidate-dict

Reply:
Domstring is a character array with 255 length:

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/30f52d18e647747de1ec29ad6f6d3f62128f5494/src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h#L62-L73

Reply:
But, since the URL length was changed in another spot in this PR:

```
    CHAR url[MAX_ICE_CONFIG_URI_LEN + 1];   //!< STUN/TURN server URL
```

We should remain consistent with the URL length

Reply:
Per offline convo, moving all such member size/type changes to a separate PR.